apps/api/.env.local.example

```
@@ -40,8 +34,6 @@ NEXT_PUBLIC_CHANNEL=local
 # Enable "Maps JavaScript API" for your project.
 NEXT_PUBLIC_GOOGLE_API_KEY=your-google-maps-api-key-here
 
-NEXT_PUBLIC_GA_MEASUREMENT_ID=****
-
 # Comma-separated list of webhook URLs to notify on map data changes (optional)
 # Format: https://webhook1.com/endpoint,https://webhook2.com/endpoint
 NOTIFY_WEBHOOK_URLS_COMMA_SEPARATED=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)